faq.html

```diff
@@ -70,7 +70,7 @@ <h4 class="panel-title">
             </div>
             <div id="G-4" class="panel-collapse collapse">
               <div class="panel-body">
-                Our builds are completely ad-free and open source! You can even go over to <a href="https://github.com/AntennaPod/AntennaPod">Github</a>
+                Our builds are completely ad-free and open source! Get them at <a href="https://play.google.com/store/apps/details?id=de.danoeh.antennapod">Google Play</a> or <a href="https://f-droid.org/repository/browse/?fdid=de.danoeh.antennapod">F-Droid</a>. You can even go over to <a href="https://github.com/AntennaPod/AntennaPod">Github</a>
```

Comment:
I know, but if you're checking this question, I guess a) you probably didn't know about the official channels and b) you probably want to get a different install to get rid of the advertisements.